templates/deno/package.json

```diff
@@ -1,32 +1,20 @@
 {
-  "name": "remix-app-template-js",
-  "private": true,
-  "description": "",
-  "license": "",
-  "sideEffects": false,
-  "scripts": {
-    "postinstall": "remix setup deno",
-    "build": "cross-env NODE_ENV=production remix build",
-    "dev:remix": "cross-env NODE_ENV=development remix watch",
-    "dev:deno": "cross-env NODE_ENV=development deno run --watch --allow-net --allow-read --allow-env ./build/index.js",
-    "dev": "cross-env NODE_ENV=development remix build && run-p dev:*",
-    "start": "cross-env NODE_ENV=production deno run --allow-net --allow-read --allow-env ./build/index.js"
-  },
-  "dependencies": {
-    "@remix-run/deno": "^1.2.3",
-    "@remix-run/react": "^1.2.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "remix": "^1.2.3"
-  },
-  "devDependencies": {
-    "@remix-run/dev": "^1.2.3",
-    "@remix-run/eslint-config": "^1.2.3",
-    "cross-env": "^7.0.3",
-    "eslint": "^8.9.0",
-    "npm-run-all": "^4.1.5"
-  },
-  "engines": {
-    "node": ">=14"
+    "private": true,
+    "sideEffects": false,
+    "scripts": {
+      "build": "cross-env NODE_ENV=production remix build",
+      "start": "cross-env NODE_ENV=production deno run --allow-net --allow-read --allow-env ./build/index.js",
+      "dev": "cross-env NODE_ENV=development remix build && run-p dev:*",
+      "dev:remix": "cross-env NODE_ENV=development remix watch",
+      "dev:deno": "cross-env NODE_ENV=development deno run --watch --allow-net --allow-read --allow-env ./build/index.js"
+    },
+    "devDependencies": {
+      "@remix-run/dev": "^1.2.2",
```

Comment:
TODO: We need to update this to be the version for the release of `@remix-run/dev` that includes these changes. @chaance do we know if this will be 1.2.3 or 1.3.0?

Reply:
I guess we can say `^1.2.4` and it will work even if we go straight from `1.2.3` to `1.30`. Feels a little weird to reference a non-existing version but 🤷 it would work.

Reply:
I think this should be `*` for now. The install script finds all remix deps in package.json and updates them with the right version when it uses the template.

Reply:
@mcansh changed this to actual values in #2189